fuel-client/src/client/schema/snapshots/fuel_gql_client__client__schema__tx__tests__dry_run_tx_gql_output.snap

```diff
@@ -1,12 +0,0 @@
----
-source: fuel-client/src/client/schema/tx.rs
```

Comment:
shouldn't new snapshot tests be generated ?

Reply:
I replaced the new one locally with the old one, so I smoked it. The one here is actually new. I didn't know that I could do that with `cargo insta`.